.travis.yml

```diff
@@ -2,7 +2,8 @@ language: node_js
 node_js:
   - 10
 before_install:
-  - curl -o factorio.tar.gz -L https://www.factorio.com/get-download/latest/headless/linux64
+  # For some reason certificate verification just randomly decided to fail :/
+  - curl --insecure -o factorio.tar.gz -L https://www.factorio.com/get-download/latest/headless/linux64
```

Comment:
They had a certificate expire, they updated them. It should be fine now. Or is it a random recurring thing that makes no sense?